assets/css/styles.css

```diff
@@ -1 +1,15 @@
+.text-center{
+	display: flex;
+	align-items: center;
+	justify-content: center;
+	margin:10px auto;
+font-size:30px;
```

Comment:
can you format this font-size: 30px; code bcs it is not with the top codes @joe-sida

Reply:
do this also @joe-sida

Reply:
sir, could you please clarify what exactly you want me to do? What do you mean by format the font-size

Reply:
what is the code editor are you using? @joe-sida

Reply:
i am using sublime for this project

Reply:
OS? @joe-sida

Reply:
windows

Reply:
windows 10

Reply:
Ok, select the code (css code) and press Ctrl+Shift+P it will format the code. After that add and commit changes @joe-sida

Reply:
No need @joe-sida it's ok

Reply:
done